plugin/trino-exasol/src/test/java/io/trino/plugin/exasol/TestExasolConnectorTest.java

```diff
@@ -126,6 +128,27 @@ protected TestTable createTableWithUnsupportedColumn()
                 "(one NUMBER(19), two GEOMETRY, three VARCHAR(10 CHAR))");
     }
 
+    @Override
+    // Override, because read-only Exasol connector does not support CREATE TABLE and INSERT statements
+    protected TestTable newTrinoTable(String namePrefix, @Language("SQL") String tableDefinition, List<String> rowsToInsert)
+    {
+        return new TestTable(exasolServer.getSqlExecutor(), TEST_SCHEMA + "." + namePrefix, tableDefinition, rowsToInsert);
+    }
+
+    @Override
+    // Override to add test schema prefix to the name of the nation table
+    protected String getNationLowerCaseTableDefinition()
+    {
+        return "AS SELECT nationkey, lower(name) name, regionkey FROM %s.nation".formatted(TEST_SCHEMA);
```

Comment:
Could you explain why do we need to specify the schema name even though the default schema is tpch in ExasolQueryRunner?